lib/streamlit/commands/page_config.py

```diff
@@ -104,7 +104,7 @@ def set_page_config(
             width=-1,  # Always use full width for favicons
             clamp=False,
             channels="RGB",
-            output_format="JPEG",
+            output_format="PNG",
```

Comment:
Hm, I think we may want to set `output_format="auto"` here, which should normally do a conversion to JPEG, but will keep the image as a PNG if it could have an alpha channel

Reply:
Thank you for the suggestion. I tested `"auto"` locally and it does work as expected!